content/locales/en.ts

```diff
@@ -43,6 +43,13 @@ const identitiesInputStrings = {
   phoneInputLabel: "Phone number *",
 };
 
+const passwordInputsStrings = {
+  newPasswordPlaceholder: "might-complex-password-42",
+  confirmNewPasswordPlaceholder: "might-complex-password-42",
+  newPasswordLabel: "New password",
+  confirmNewPasswordLabel: "Confirm new password",
+};
+
```

Comment:
Hm, not sure we want to move this out as it will only be used in one path. What do you think?

Reply:
You forgot to do it for `fr.ts` :).